.pack.mk

```diff
@@ -22,17 +22,26 @@ ifeq (${VERSION}, $(filter ${VERSION}, 1.10))
 endif
 
 prepare:
-	rm -rf build packpack
-	git clone https://github.com/packpack/packpack.git
+#	rm -rf build packpack
+#	git clone https://github.com/packpack/packpack.git
 
 package: prepare
 	if [ "${TARANTOOL_SERIES}" != "1.10" ]; then \
 		if [ -n "${GIT_TAG}" ]; then \
 			export VERSION="$$(echo ${GIT_TAG} | sed 's/-/~/')"; \
```

Comment:
Looks like you missed the logic for Alpine here. If there is `2.11.0-rc1` tag, we will replace `-` with `~` which is incorrect for Alpine as far as I understand. Also, we need to take into account that a tag may be `2.12.0-entrypoint` and this should be handled as well.

Reply:
fixed

Reply:
Nope. Please resolve it.

Reply:
done